usage-collection/src/main/proto/usage.proto

```diff
@@ -1168,3 +1172,52 @@ message CDPUDXBundleDetails {
   // The type of the diagnostics bundle.
   string type = 5;
 }
+
```

Comment:
Is the timestamp of these events captured automatically?
From looking at LoggingUsageReporter - it looks like some events set it explicitly, while others don't (guessing the rest have some kind of a timestamp within the event itself?)

Reply:
timestamp is captured automatically